lego/apps/events/models.py

```diff
@@ -518,6 +518,11 @@ def spots_left_for_user(self, user):
 
         return sum([pool.spots_left() for pool in pools])
 
+    @property
+    def is_abakom_only(self):
```

Comment:
I see your point here, making the event only visible for abakom if abakom is in can_view_groups - but couldnt we just have a `is_abakom` boolean flag on the event model instead? That way we wont rely on the can_view_groups and we can filter directly on it. Have you thought of that?